mlrun/datastore/targets.py

```diff
@@ -556,7 +556,7 @@ def write_dataframe(
                             break
                 # Partitioning will be performed on timestamp_key and then on self.partition_cols
                 # (We might want to give the user control on this order as additional functionality)
-                partition_cols += self.partition_cols or []
+                partition_cols = (self.partition_cols or []) + partition_cols
```

Comment:
why do we need to change this? don't we create the partition dirs "manually"?

Reply:
We don't need it (old leftovers) :) 
I'll remove it